runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -24,6 +24,7 @@
 #include "circle_loader.h"
 #include "tflite_loader.h"
 #include "trix_loader.h"
+#include "traininfo_loader.h"
```

Comment:
(Out of curiosity) Why do you add `#include` here? It is not the end of `#include`, and it is not in alphabetical order. If you intended almost-alphabetical order, it would be better to put before `trix_loader.h`.